virttest/qemu_vm.py

```diff
@@ -3895,7 +3895,10 @@ def activate_nic(self, nic_index_or_name):
             pcie = False
         else:
             pcie = nic['nic_model'] not in ['e1000', 'rtl8139']
-        bus_spec = self._get_pci_bus(nic_params, 'nic', pcie)
+        if self.params.get("machine_type").startswith('s390'):
+            bus_spec = {'aobject': 'virtual-css'}
```

Comment:
@fbq815 @PaulYuuu are other kinds of devices in s390 required to be attached to the `virtual-css` bus as well? if so I'd suggest introducing a method for this procedure.
```python
    def _get_bus(self, params, *args, **kwargs):
        if params.get("machine_type") ...
        ...
```

Reply:
In fact, @fbq815 introduced a method before, it named `_get_css_bus` and returns `virtual-css` directly so I think it isn't necessary.
I am not sure if other devices need `virtual-css` as well, so a new method is added to return pci bus and other types of bus seems not mandatory.

Reply:
other devices seems could use other buses, e.g. virtio-blk-ccw could attach to virtio-bus instead of virtual-css, but virtio-net-ccw could not use it

Reply:
after investigation, all devices on s390x will use virtual-css to be the parent bus, virtio-bus should be a pseudo bus